README.md

```diff
@@ -1,18 +1,24 @@
 # Rise Page Web Component
 
 ## Introduction
-`rise-page` is a web component that controls the playback of its child components, telling them when to play, pause or stop. Additionally, it surfaces methods for returning details about the Display.
+`rise-page` is a web component that controls the playback of its child playlists. Playlist items that have the same duration and are located in the same position within a playlist as another playlist item within a separate playlist, will have their playback synchronized.
```

Comment:
@donnapep Ya it is hard to explain that in a concise way. The demo is a good reference for understanding that, what about referencing the demo? So maybe add:

"Please see `demo.html` for an example and understanding of playback synchronization".